CONTRIBUTING.md

```diff
@@ -27,6 +27,13 @@ The following commands are available in the Makefile:
 - `make test` runs the tests to ensure that all functionality is working as intended.
 - `make proto` regenerates the corresponding code if you have made any changes to the proto buffer files.
 
+Help and description messages (example: CLI subcommands descriptions) MUST follow this style:
```

Comment:
I suggest:

```
## Guidelines for CLI Help Messages:

- Start all messages with a lowercase letter.
- Avoid stating defaults in the help string, as Cobra automatically adds them.
- Include a range for flags that accept a range of values.
- Exclude parentheses "()" from help messages.
```

Reply:
I agree, a better way to write that. commited.